src/main/java/Duke.java

```diff
@@ -1,39 +1,57 @@
 import java.util.Scanner;
+import java.util.ArrayList;
 
 public class Duke {
     private static String line = "__________________________________________________________";
+    private static Task[] inputList = new Task[101];
+    private static int length = 0;
```

Comment:
Variable name length might be confusing as it might refer to length of a string, consider using something else like numberOfTasks for example